ludwig/features/text_feature.py

```diff
@@ -261,6 +264,34 @@ def get_output_dtype(cls):
     def output_shape(self) -> torch.Size:
         return torch.Size([self.decoder_obj.config.max_sequence_length])
 
+    def update_metrics(
+        self,
+        targets: Tensor,
+        predictions: Dict[str, Tensor],
+        decoded_targets: Optional[List[str]] = None,
+        decoded_predictions: Optional[List[str]] = None,
+    ) -> None:
+        """Updates metrics with the given targets and predictions.
+
+        If decoded_targets and decoded_predictions are provided, as through LLM model types, then additional
+        response-based metrics like BLEU and ROUGE are also computed.
+
+        Args:
+            targets: Tensor with target values for this output feature.
+            predictions: Dict of tensors returned by predictions().
+        """
+        for metric_name, metric_fn in self._metric_functions.items():
+            prediction_key = get_metric_tensor_input(metric_name)
+            if prediction_key != RESPONSE:
```

Comment:
nit, i'd rather have more explitly the response case and the else case than the non response case

Reply:
Done.